src/Util.hs

```diff
@@ -155,20 +157,30 @@ checkReleaseExists release = do
     error ("Release " <> T.unpack release <> " does not exist in release store at " <> T.unpack (bs_label bs ))
   return ()
 
-loadMergedContext :: ToolConfig -> IO JS.Value
-loadMergedContext tcfg = do
-  let cacheDir = T.unpack (tc_contextCache tcfg)
-
-  values <- for (M.toList (SM.toMap (tc_configSources tcfg))) $ \name_src -> do
-    let name = fst name_src
-    let src = snd name_src
+-- Load template context from the infrastructure
+loadMergedInfrastructureContext :: ToolConfig -> IO (HM.HashMap T.Text JS.Value)
+loadMergedInfrastructureContext tcfg = do
+  values <- for (M.toList (SM.toMap (tc_configSources tcfg))) $ \(name,src) -> do
     let cacheFilePath = configContextCacheFilePath tcfg name
     lbs <- LBS.readFile cacheFilePath
     case JS.eitherDecode' lbs of
      (Left e) -> error ("Unable to parse json from " <> cacheFilePath)
      (Right jv) -> return (takeBaseName cacheFilePath, jv)
 
-  return (JS.Object (HM.fromList ([(T.pack file,jv) | (file,jv) <- values]) ))
+  return (HM.fromList ([(T.pack file,jv) | (file,jv) <- values]) )
+
+-- Load template context from the release package
+loadMergedReleaseContext :: FilePath -> ReleaseConfig -> IO (HM.HashMap T.Text JS.Value)
+loadMergedReleaseContext dir rcfg = do
+
+  values <- for (M.toList (SM.toMap (rc_configSources rcfg))) $ \(name,src) -> do
+    let filepath = dir </> T.unpack src
```

Comment:
Guessing this implies no leading '/' in the config var? Does this mean (if I wanted) I could also go ../ (not that I'd want to, nor do I think it's a security issue - just trying to understand the code)

Reply:
A leading / in the path is not intended, but will work, as per docs for [`</>`](
https://hackage.haskell.org/package/filepath-1.4.2.1/docs/System-FilePath-Posix.html#v:-60--47--62-
)
`..` would also work.